go/mysql/binlog_event_make.go

```diff
@@ -296,9 +296,9 @@ func NewTableMapEvent(f BinlogFormat, s *FakeBinlogStream, tableID uint64, tm *T
 		1 + // table name length
 		len(tm.Name) +
 		1 + // [00]
-		1 + // column-count FIXME(alainjobart) len enc
+		lenEncIntSize(uint64(len(tm.Types))) + // column-count len enc
```

Comment:
Looks like `lenEncIntSize` expects a uint64 and hence the type coercion.
Do you think it's worth putting in a comment line for future readers/editors?

Reply:
Not really, if the coercion is not there, the code does not compile so it seems like a comment is about then something the compiler already enforces as well?